<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PKResultWidget</class>
 <widget class="QWidget" name="PKResultWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>266</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
border-image: url(:/image/PKResultUI/bg_gameover.png);}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="winnerTextLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 12px &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="waitSelectedTextLabel">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>60</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: bold 14px &quot;宋体&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14px; font-weight:600; color:#fffffd;&quot;&gt;恭喜你获得胜利，想要我做&lt;br/&gt;什么呢？  &lt;/span&gt;&lt;span style=&quot; color:#f27ef6;&quot;&gt;60s后将自动选择&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="RankListWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>281</width>
      <height>131</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);
font:bold 12px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QWidget" name="RewardBackGroundWidget" native="true">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>45</y>
      <width>215</width>
      <height>210</height>
     </rect>
    </property>
    <widget class="QWidget" name="RewardBackGroundMask" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>215</width>
       <height>210</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="RewardBackGround" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>215</width>
       <height>210</height>
      </rect>
     </property>
    </widget>
    <zorder>RewardBackGround</zorder>
    <zorder>RewardBackGroundMask</zorder>
   </widget>
   <widget class="QTableWidget" name="RewardTableWidget">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>100</y>
      <width>180</width>
      <height>150</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="autoScroll">
     <bool>false</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>false</bool>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QLabel" name="seclectedLabel">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>70</y>
      <width>82</width>
      <height>82</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="seclectedTipLabel">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>167</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 12px &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>恭喜你获得了亲一口，快让主播为&lt;br/&gt;你表演吧~</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitButton">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>211</y>
      <width>90</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/PKResultUI/bt_gameover.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/PKResultUI/icon_jiangbei.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="adviceBtn">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>11</y>
      <width>18</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image: url(:/image/HostHomeUI/bt_useradvice_hover.png);}
QPushButton{border-image: url(:/image/HostHomeUI/bt_useradvice.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closebtn">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image: url(:/image/PKResultUI/bt_close_hover.png);}
QPushButton{border-image: url(:/image/PKResultUI/bt_close.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="minBtn">
    <property name="geometry">
     <rect>
      <x>534</x>
      <y>7</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{border-image:
url(:/image/PKResultUI/bt_min_hover.png);}
QPushButton{border-image: url(:/image/PKResultUI/bt_min.png);}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>RewardBackGroundWidget</zorder>
   <zorder>seclectedLabel</zorder>
   <zorder>winnerTextLabel</zorder>
   <zorder>waitSelectedTextLabel</zorder>
   <zorder>RankListWidget</zorder>
   <zorder>seclectedTipLabel</zorder>
   <zorder>label_2</zorder>
   <zorder>adviceBtn</zorder>
   <zorder>RewardTableWidget</zorder>
   <zorder>quitButton</zorder>
   <zorder>closebtn</zorder>
   <zorder>minBtn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
